tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -416,7 +416,7 @@ def _tiered_storage_warmup(self, scale, topic_name):
         # such that actual segment size is as close as possible to the desired
         # size. In default configuration we run with `log_segment_size_jitter_percent=5`
         # adjust the messages size to always be larger than that.
-        warmup_message_size = math.ceil(scale.segment_size * 0.06)
+        warmup_message_size = math.ceil(scale.segment_size * 0.5)
```

Comment:
So in along the lines of the comment above does this usually result in segments 1.5x the larger size? Since 2 messages gives you 100% of the non-jitter size which will almost [1] always be below the jitterred target (100% to 105%), leading to 3 message segments?

In the (ambiguous) spirit of the comment, maybe we should use 0.53 since 1.06 jumps over the jittered range.

[1] Only "almost" not "always" because of random config batches, I think.

Reply:
Thanks for your comment above, I think understand now what it's trying to say (though the motivation seems slightly bogus to me).

Above you write:

>  if the segment seize is 100, the jitter range is 100-105

From what I can tell we jitter both ways so the range is 95-105 which I think it makes it not "almost always" but rather 50% but the reasoning to bump it to 0.53 still seems sound to me so I will make the change. 1.06 also sounds fine to me but from my testing there wasn't a big difference anymore at that point.

Reply:
> From what I can tell we jitter both ways so the range is 95-105

That was also my recollection, but I was just assuming the comment to be correct. Perhaps just replace the comment now with one of your own given the change?

+1 on the rest.